### PR TITLE
Recommendation for ssh and samba add-ons

### PR DESCRIPTION
**Pull request by cogneato**
Made a stronger recommendation for the ssh addon and samba addon following an install of Hassio.

**Description:**

**Pull request in home-assistant (if applicable):** home-assistant/home-assistant#<home-assistant PR number goes here>

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next Home Assistant release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

**Review by DubhAd — DISMISSED** [review]
Thanks. Given the number of folks that don't install the SSH add-on and get themselves in a pickle, I think it's important this is more strongly worded.

File: source/hassio/installation.markdown [hunk 1]
```diff
@@ -49,7 +49,7 @@ The following will take you through the steps required to install Hass.io.
 
 6. You will be able to reach your installation at `http://hassio.local:8123` (if your router supports mDNS, otherwise see below).
 
-7. Enable either the [Samba add-on][samba] or the [SSH add-on][ssh] to manage your configuration in `/config/` (From the UI choose **Hass.io** which is located in the sidebar).
+7. It is important to provide yourself proper access and the Hassio custom cli tools. Both the [Samba add-on][samba] and the [SSH add-on][ssh] should be the first add-ons a user should install before making changes to configuration in `/config/` (From the UI choose **Hass.io** which is located in the sidebar and then the Add-on store).
```
Comment (frenck) [inline source/hassio/installation.markdown]:
```suggestion
7. It is important to provide yourself proper access, including the Hass.io CLI tools. Both the [Samba add-on][samba] and the [SSH add-on][ssh] should be the first add-ons you should install, before making changes to the configuration in the `/config/` folder. From the UI choose **Hass.io**, which is located in the sidebar and then the add-on store.
```

**Review by frenck — APPROVED** [review]
Tiny suggestion to improve the language

**Review by frenck — CHANGES_REQUESTED** [review]
Arrg wrong button, let me try again

File: source/hassio/installation.markdown (hunk 1 above)
Comment (frenck) [reply source/hassio/installation.markdown]: The only thing that is missing IMHO, is WHY one needs this.
However, still a major improvement, so 👍

**Review by frenck — APPROVED** [review]
